src/iOS/Xamarin.iOS/Samples/Geometry/SpatialOperations/SpatialOperations.cs

```diff
@@ -164,15 +165,15 @@ public override void ViewDidLoad()
         public override void LoadView()
         {
             // Create the views.
-            View = new UIView() { BackgroundColor = UIColor.White };
+            View = new UIView() { BackgroundColor = ApplicationTheme.BackgroundColor };
 
             _myMapView = new MapView();
             _myMapView.TranslatesAutoresizingMaskIntoConstraints = false;
 
             _operationChoiceButton = new UISegmentedControl("Difference", "Intersection", "Symm. diff.", "Union")
             {
-                BackgroundColor = UIColor.White,
-                TintColor = UIColor.White,
```

Comment:
With the new UISegmentedControl design, I don't think tint color needs to be set at all. I just tried commenting out that line and the appearance doesn't change at all.